src/Views/FirmwareView.vala

```diff
@@ -120,12 +120,12 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
                 add_device (devices[i]);
             }
 
-            placeholder_alert_view.title = _("Firmware Updates Are Not Available");
-            placeholder_alert_view.description = _("Firmware updates are not supported on this or any connected devices.");
+            placeholder_alert_view.title = _("No new firmware available");
+            placeholder_alert_view.description = _("None of your devices support installing new firmware this way.");
```

Comment:
I like the attempt at using more natural language here, but "this way" seems vague and kind of implies an unanswered question of there being another way to install updates

Reply:
Yes, you can connect to pins and do it the old way.
Will change.

Reply:
```suggestion
            placeholder_alert_view.description = _("Nothing on or connected to your system supports installing new firmware this way.");
```